lib/service/service.go

```diff
@@ -4194,6 +4196,10 @@ func validateConfig(cfg *Config) error {
 		cfg.Console = io.Discard
 	}
 
+	if cfg.Log == nil {
+		return trace.BadParameter("config: please supply logger")
```

Comment:
Why can't we just create a logger if one was not provided?

Reply:
What should the default logger be in that situation? Currently we just panic at the first use of `cfg.Log` or `process.log`.

Reply:
We do this in several places:

```
	if cfg.Log == nil {
		logger = log.StandardLogger()
	}
```